default/templates/default/base-uploader.html

```diff
@@ -15,8 +15,16 @@
     <b>{% trans "Success!" %}</b> result.zip(<span class="file-size"></span>)
     <a href="javascript:void(0)" class="download" target="_blank">{% trans "Download" %}</a>
 </div>
+<div class="response-warning action-failed alert alert-warning hidden">
+    <ul></ul>
+</div>
+<div class="response-warning file-process-failed alert alert-warning hidden">
+    {% trans "Some of the files could not be validated." %}
+    {% trans "You can add more files or remove them, then click on <b>Start</b> again." %}
+    {% trans "Invalid files will be ignored by default." %}
```

Comment:
Can we be clearer on what we mean? e.g.

"We couldn't check that all files were OCDS JSON."
…
"Files that are not OCDS JSON will be ignored by default."

Reply:
ok!

Reply:
Also, I couldn't find where the logic is to ignore non-OCDS files?

Reply:
I guessed an user would like to exclude an invalid file (invalid means that it is not JSON, or not a release/record package in places a package is expected). Invalid JSON files and non-package files can crash the operations to make, that's why I think they should be ignored. We can remove this behaviour and let the user choose.

To determine a file is OCDS or not involves CoVE validation I think? Is this something we will like to do inside Toucan?

Reply:
I think it's okay to ignore the files, as long as the user sees which files were ignored. Since this PR has been open a while, we can open a new issue to explore that.

OCDS Kit can work with data that doesn't pass CoVE validation, so I don't think we need to do that extra validation. I think checking for the presence of dict keys is enough for Toucan.

Reply:
Yep, I thought that just parsing and checking some keys would be fine. The user will see a message for each file that has an issue as in the image.

![screenshot-127 0 0 1_8000-2019 12 19-15_13_28](https://user-images.githubusercontent.com/1881108/71198288-33dda000-2272-11ea-9c48-94266a6770bf.png)

Reply:
Looks great!